maritalk/model.py

```diff
@@ -0,0 +1,34 @@
+import requests
+
+class MariTalk:
+    def __init__(self, key: str, host: str='https://chat.maritaca.ai/api'):
+        self.key = key
+        self.host = host
+
+    def generate(self, messages: str, temperature: float=0.7, top_p: float=0.95, max_tokens: int=512, stopping_tokens: list=[]):
```

Comment:
Type hinting with built-in types was [introduced in python 3.9](https://docs.python.org/3/whatsnew/3.9.html#type-hinting-generics-in-standard-collections), so we would be dropping support for older python versions in this case. I think we could just `from typing import List` and:

```suggestion
    def generate(self, messages: str, temperature: float=0.7, top_p: float=0.95, max_tokens: int=512, stopping_tokens: List=[]):
```

Reply:
Good ideia